board/forms.py

```diff
@@ -0,0 +1,24 @@
+from django import forms
+from .models import Person, City
+
+
+class PersonForm(forms.ModelForm):
+    class Meta:
+        model = Person
+        fields = ('name', 'birthdate', 'country', 'city',
+                  'job', 'income', 'expenses', 'kids')
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['city'].queryset = City.objects.none()
+
+        if 'country' in self.data:
+            try:
+                country_id = int(self.data.get('country'))
+                self.fields['city'].queryset = City.objects.filter(
+                    country_id=country_id).order_by('name')
+            except (ValueError, TypeError):
+                pass  # invalid input from the client; ignore and fallback to empty City queryset
+        elif self.instance.pk:
+            self.fields['city'].queryset = self.instance.country.city_set.order_by(
+                'name')
```

Comment:
Setting the city dropdown in accordance to the country chosen in `self.data` of the form